### PR TITLE
Patch configuration alias bug

### PR DESCRIPTION
**Pull request by dstallenberg**

### CODE REVIEW

File: packages/core/lib/Configuration.ts [hunk 1]
```diff
@@ -284,7 +284,7 @@ export class Configuration {
             type: "number",
           },
           "evaluation-budget": {
-            alias: ["e"],
+            alias: [],
```
Comment (mitchellolsthoorn) [inline packages/core/lib/Configuration.ts:287]: Why was this a bug?

Comment (dstallenberg) [reply packages/core/lib/Configuration.ts:287]: I accidentally used "e" twice (also for exclude)
So it was overwritten.

Comment (mitchellolsthoorn) [reply packages/core/lib/Configuration.ts:287]: We probably also only need an alias for the main budget: totalTime

Comment (dstallenberg) [reply packages/core/lib/Configuration.ts:287]: Created an issue for this.

**Review by mitchellolsthoorn — APPROVED** [review]

**Pull request merged**